bundles/ranvier-commands/commands/wear.js

```diff
@@ -13,33 +14,63 @@ module.exports = (srcPath) => {
       arg = arg.trim();
 
       if (!arg.length) {
-        return Broadcast.sayAt(player, 'Wear what?');
+        return say(player, 'Wear what?');
       }
 
       const item = Parser.parseDot(arg, player.inventory);
 
       if (!item) {
-        return Broadcast.sayAt(player, "You aren't carrying anything like that.");
+        return say(player, "You aren't carrying anything like that.");
       }
 
       if (!item.slot) {
-        return Broadcast.sayAt(player, `You can't wear ${item.display}.`);
+        return say(player, `You can't wear ${item.display}.`);
+      }
+
+      if (item.level > player.level) {
+        return say(player, "You can't use that yet.");
       }
 
       try {
         player.equip(item);
       } catch (err) {
         if (err instanceof EquipSlotTakenError) {
           const conflict = player.equipment.get(item.slot);
-          return Broadcast.sayAt(player, `You will have to remove ${conflict.display} first.`);
+          return say(player, `You will have to remove ${conflict.display} first.`);
```

Comment:
Might be nice if there was a switch to auto-replace any worn equipment when equipping new eq.